org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/parser/CompSym.java

```diff
@@ -1,7 +1,7 @@
 
 //----------------------------------------------------
 // The following code was generated by CUP v0.11a beta 20060608
-// Sun Apr 08 09:54:30 PDT 2018
+// Mon Dec 09 00:19:54 CET 2019
```

Comment:
Please do not commit these files unless you change them. The date change sucks but they should not be committed unless they have a semantic change

Reply:
Cleaned 🙆‍♂